lib/Echidna/UI.hs

```diff
@@ -126,10 +126,11 @@ ui :: ( MonadCatch m, MonadRandom m, MonadReader x m, MonadUnliftIO m
    -> m Campaign
 ui v w ts d = let xfer e = use hasLens >>= \c -> isDone c >>= ($ e c) . bool id forever in do
   s <- (&&) <$> isTerminal <*> view (hasLens . dashboard)
+  g <- view (hasLens . to seed)
   c <- if s then do bc <- liftIO $ newBChan 100
                     t <- forkIO $ campaign (xfer $ liftIO . writeBChan bc) v w ts d >> pure ()
                     a <- monitor (killThread t)
-                    liftIO (customMain (mkVty defaultConfig) (Just bc) a mempty)
+                    liftIO (customMain (mkVty defaultConfig) (Just bc) a $ Campaign mempty mempty (fromJust d))
```

Comment:
Can we avoid using `fromJust`?

Reply:
the `fromJust` is for the `Maybe GenDict` parameter and was not my change